src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -49,6 +49,6 @@ public int getNumber(){
     @Override
     public int compareTo(LottoNumber lottoNumber) {
         LottoNumber compare = (LottoNumber) lottoNumber;
-        return this.getNumber() - compare.getNumber();
+        return this.getNumber() - compare.number;
```

Comment:
아래 처럼 사용하면 여기서도 `getNumber()`를 지울 수 있겠네요 :)
```suggestion
        return this.number - compare.number;
```